### PR TITLE
Convert .md files to .rst

### PR DESCRIPTION
**Pull request by lucianomartin**
Part of https://xmosjira.atlassian.net/browse/AP-270

### CODE REVIEW

File: examples/ffva/README.rst
Comment (mbanth) [inline examples/ffva/README.rst:1]: Missed header line at line 47.

File: examples/speech_recognition/README.rst
Comment (mbanth) [inline examples/speech_recognition/README.rst:1]: Missed header lines at lines 7 and 72

**Review by mbanth — CHANGES_REQUESTED** [review]

File: examples/ffva/README.rst
Comment (lucianomartin) [reply examples/ffva/README.rst:1]: Fixed.

File: examples/speech_recognition/README.rst
Comment (lucianomartin) [reply examples/speech_recognition/README.rst:1]: Fixed.

**Review by mbanth — APPROVED** [review]
Looks good!

**Pull request merged**